sympy/printing/tests/test_mathml.py

```diff
@@ -1182,3 +1183,45 @@ def test_print_Indexed():
     assert mathml(IndexedBase(a),printer = 'presentation') == '<mi>a</mi>'
     assert mathml(IndexedBase(a/b),printer = 'presentation') == '<mrow><mfrac><mi>a</mi><mi>b</mi></mfrac></mrow>'
     assert mathml(IndexedBase((a,b)),printer = 'presentation') == '<mrow><mfenced><mi>a</mi><mi>b</mi></mfenced></mrow>'
+
+
+def test_print_Vector():
+    ACS = CoordSys3D('A')
+    assert mathml(Cross(ACS.i, ACS.j*ACS.x*3 + ACS.k), printer='presentation') == \
+    '<mrow><msub><mover><mi mathvariant="bold">i</mi><mo>^</mo></mover><mi mathvariant="bold">A</mi></msub><mo>&#xD7;</mo><mfenced><mrow><mfenced><mrow><mn>3</mn><mo>&InvisibleTimes;</mo><msub><mi mathvariant="bold">x</mi><mi mathvariant="bold">A</mi></msub></mrow></mfenced><mo>&InvisibleTimes;</mo><msub><mover><mi mathvariant="bold">j</mi><mo>^</mo></mover><mi mathvariant="bold">A</mi></msub><mo>+</mo><msub><mover><mi mathvariant="bold">k</mi><mo>^</mo></mover><mi mathvariant="bold">A</mi></msub></mrow></mfenced></mrow>'
```

Comment:
`strlines` will convert this to
```
(
'<mrow><msub><mover><mi mathvariant="bold">i</mi><mo>^</mo></mo'
'ver><mi mathvariant="bold">A</mi></msub><mo>&#xD7;</mo><mfence'
'd><mrow><mfenced><mrow><mn>3</mn><mo>&InvisibleTimes;</mo><msu'
'b><mi mathvariant="bold">x</mi><mi mathvariant="bold">A</mi></'
'msub></mrow></mfenced><mo>&InvisibleTimes;</mo><msub><mover><m'
'i mathvariant="bold">j</mi><mo>^</mo></mover><mi mathvariant="'
'bold">A</mi></msub><mo>+</mo><msub><mover><mi mathvariant="bol'
'd">k</mi><mo>^</mo></mover><mi mathvariant="bold">A</mi></msub'
'></mrow></mfenced></mrow>'
)
```
Then you can just write
```
assert ... == (
    ...
    ...
    )
```
Here it is (but it is not in misc.py yet).
```python
def strlines(s, c=64):
    """Return a cut-and-pastable string that, when printed, is
    equivalent to the input.  The lines will be surrounded by
    parentheses and no line will be longer than c (default 64)
    characters. If the line contains newlines characters, the
    `rawlines` result will be returned.

    Examples
    ========

    >>> from sympy.utilities.misc import strlines
    >>> q = 'this is a long string that should be broken into shorter lines'
    >>> print(strlines(q, 40))
    (
    'this is a long string that should be b'
    'roken into shorter lines'
    )
    >>> q == (
    ... 'this is a long string that should be b'
    ... 'roken into shorter lines'
    ... )
    True

    See Also
    ========
    filldedent, rawlines
    """
    if type(s) not in string_types:
        raise ValueError('expecting string input')
    if '\n' in s:
        return rawlines(s)
    q = '"' if repr(s).startswith('"') else "'"
    q = (q,)*2
    if '\\' in s:  # use r-string
        m = '(\nr%s%%s%s\n)' % q
        j = '%s\nr%s' % q
        c -= 3
    else:
        m = '(\n%s%%s%s\n)' % q
        j = '%s\n%s' % q
        c -= 2
    out = []
    while s:
        out.append(s[:c])
        s=s[c:]
    return m % j.join(out)
```

And here is the diff that I would suggest:
```diff
diff --git a/sympy/utilities/misc.py b/sympy/utilities/misc.py
index 6b35358..47585dd 100644
--- a/sympy/utilities/misc.py
+++ b/sympy/utilities/misc.py
@@ -7,7 +7,8 @@
 import re as _re
 import struct
 from textwrap import fill, dedent
-from sympy.core.compatibility import get_function_name, range, as_int
+from sympy.core.compatibility import (get_function_name, range, as_int,
+    string_types)



@@ -24,13 +25,67 @@ def filldedent(s, w=70):

     Empty line stripping serves to deal with docstrings like this one that
     start with a newline after the initial triple quote, inserting an empty
-    line at the beginning of the string."""
+    line at the beginning of the string.
+
+    See Also
+    ========
+    strlines, rawlines
+    """
     return '\n' + fill(dedent(str(s)).strip('\n'), width=w)


+def strlines(s, c=64):
+    """Return a cut-and-pastable string that, when printed, is
+    equivalent to the input.  The lines will be surrounded by
+    parentheses and no line will be longer than c (default 64)
+    characters. If the line contains newlines characters, the
+    `rawlines` result will be returned.
+
+    Examples
+    ========
+
+    >>> from sympy.utilities.misc import strlines
+    >>> q = 'this is a long string that should be broken into shorter lines'
+    >>> print(strlines(q, 40))
+    (
+    'this is a long string that should be b'
+    'roken into shorter lines'
+    )
+    >>> q == (
+    ... 'this is a long string that should be b'
+    ... 'roken into shorter lines'
+    ... )
+    True
+
+    See Also
+    ========
+    filldedent, rawlines
+    """
+    if type(s) not in string_types:
+        raise ValueError('expecting string input')
+    if '\n' in s:
+        return rawlines(s)
+    q = '"' if repr(s).startswith('"') else "'"
+    q = (q,)*2
+    if '\\' in s:  # use r-string
+        m = '(\nr%s%%s%s\n)' % q
+        j = '%s\nr%s' % q
+        c -= 3
+    else:
+        m = '(\n%s%%s%s\n)' % q
+        j = '%s\n%s' % q
+        c -= 2
+    out = []
+    while s:
+        out.append(s[:c])
+        s=s[c:]
+    return m % j.join(out)
+
+
 def rawlines(s):
     """Return a cut-and-pastable string that, when printed, is equivalent
-    to the input. The string returned is formatted so it can be indented
+    to the input. Use this when there is more than one line in the
+    string. The string returned is formatted so it can be indented
     nicely within tests; in some cases it is wrapped in the dedent
     function which has to be imported from textwrap.

@@ -82,6 +137,10 @@ def rawlines(s):
         'that\\n'
         '    '
     )
+
+    See Also
+    ========
+    filldedent, strlines
     """
     lines = s.split('\n')
     if len(lines) == 1:
```

Reply:
Thanks! Correct that it is still not done. I was waiting for other possible review comments. I'll try to get this sorted out in the next few days.

Reply:
see #16253